drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-backend/src/main/java/org/drools/workbench/screens/guided/dtable/backend/server/conversion/BRLColumnSubHeaderBuilderDefault.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright 2020 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.drools.workbench.screens.guided.dtable.backend.server.conversion;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.drools.workbench.models.datamodel.rule.ActionInsertFact;
+import org.drools.workbench.models.datamodel.rule.FactPattern;
+import org.drools.workbench.models.datamodel.rule.IPattern;
+import org.drools.workbench.models.guided.dtable.backend.GuidedDTDRLPersistence;
+import org.drools.workbench.models.guided.dtable.shared.model.BRLActionColumn;
+import org.drools.workbench.models.guided.dtable.shared.model.BRLConditionColumn;
+import org.drools.workbench.models.guided.dtable.shared.model.BRLVariableColumn;
+import org.drools.workbench.models.guided.dtable.shared.model.DTCellValue52;
+import org.drools.workbench.models.guided.dtable.shared.model.GuidedDecisionTable52;
+import org.drools.workbench.screens.guided.dtable.backend.server.conversion.util.ColumnContext;
+
+import static org.drools.workbench.screens.guided.dtable.backend.server.conversion.util.Util.hasContent;
+
+/**
+ * Makes a single column out of the BRL column(s)
+ */
+public class BRLColumnSubHeaderBuilderDefault
+        implements BRLColumnSubHeaderBuilder {
+
+    private SubHeaderBuilder subHeaderBuilder;
+    private ColumnContext columnContext;
+    private final GuidedDecisionTable52 dtable;
+    private final Map<String, String> varListInOrder = new HashMap<>();// original, new one
+
+    public BRLColumnSubHeaderBuilderDefault(final SubHeaderBuilder subHeaderBuilder,
+                                            final ColumnContext columnContext,
+                                            final GuidedDecisionTable52 dtable) {
+        this.subHeaderBuilder = subHeaderBuilder;
+        this.columnContext = columnContext;
+        this.dtable = dtable;
+    }
+
+    @Override
+    public void brlActions(final BRLActionColumn brlColumn) {
```

Comment:
It seems that this method is doing too many things. Could you please modularize it in multiple methods with proper names?